.github/workflows/stale.yml

```
@@ -0,0 +1,30 @@
+name: Stale
+
+on:
+  schedule:
+    # run weekly on sunday
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
+
+
+jobs:
+  label:
+    name: Label issues and PRs
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 30
+          days-before-pr-stale: 30
```

```suggestion
          days-before-stale: 30
```
This has a combined option too!